tcrconvert/convert.py

```diff
@@ -18,67 +18,77 @@
 }
 
 
-def choose_lookup(frm, to, species='human'):
-    """Determine which lookup table to use and get filepath.
+def choose_lookup(frm, to, species='human', verbose=True):
+    """Choose lookup table
+
+    Determine which CSV lookup table to use based on the the input format
+    (``frm``) and returns the path to that file.
 
     :param frm: Input format of TCR data ``['tenx', 'adaptive', 'adaptivev2', 'imgt']``
     :type frm: str
     :param to: Output format of TCR data ``['tenx', 'adaptive', 'adaptivev2', 'imgt']``
     :type to: str
-    :param species: Species
+    :param species: Species, defaults to ``'human'``
     :type species: str, optional
+    :param verbose: Whether to show all messages, defaults to ``True``
+    :type verbose: bool, optional
     :return: Path to correct lookup table
     :rtype: str
 
     :Example:
 
     >>> import tcrconvert
-    >>> tcrconvert.convert.choose_lookup('imgt', 'adaptive')
+    >>> tcrconvert.convert.choose_lookup('imgt', 'adaptive', verbose=False)
```

Comment:
i think you need to add `  # doctest: +ELLIPSIS`  at the end of this line since `doctest` fails without it

```
Trying:
    tcrconvert.convert.choose_lookup('imgt', 'adaptive', verbose=False)
Expecting:
    '.../tcrconvert/data/human/lookup.csv'
**********************************************************************
File "/Users/schambe3/github/seshadrilab/tcrconvert/tcrconvert/convert.py", line 41, in convert.choose_lookup
Failed example:
    tcrconvert.convert.choose_lookup('imgt', 'adaptive', verbose=False)
Expected:
    '.../tcrconvert/data/human/lookup.csv'
Got:
    '/Users/schambe3/github/seshadrilab/tcrconvert/tcrconvert/data/human/lookup.csv'
```

see https://docs.python.org/3/library/doctest.html#directives

Reply:
Wow, I didn't know you could do that! Added in ab75385.